File: Java/1568.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    static int[] d;

    static int maxi(int a, int b) {
        int rIdx = a-1;

        for(int i = a-1; i < b; i++)
            rIdx = Main.d[i] > Main.d[rIdx] ? i : rIdx;

        return ++rIdx;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n = Integer.parseInt(scanner.nextLine());

        Main.d = Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();

        int a = scanner.nextInt();
        int b = scanner.nextInt();

        System.out.println(maxi(a, b));
    }
}

File: Python/1568.py
n, a, b, d = 0, 0, 0, []

def maxi(a:int, b:int) -> int:
    rIdx = a-1

    for i in range(a-1, b):
        global d
        rIdx = i if d[i] > d[rIdx] else rIdx

    return rIdx+1

if __name__ == "__main__":
    n = int(input())
    d = list(map(int, input().split()))
    a, b = map(int, input().split())

    print(maxi(a, b))